R/editable_fun.R

```diff
@@ -1,81 +1,104 @@
 ##' @title Generation and Checks TestHarness creation
 ##' @param package_path path to the test package
 ##' @param function_name function name in the package
-##' @description This function creates two different testharness one for generating the vectors of 
-##' user defined size that are creatd by deepstate and second for writing asserts/checks 
-##' on the result/generated inputs.
+##' @description This function creates two different testharness one for 
+##' generating the vectors of user defined size that are created by deepstate 
+##' and second for writing asserts/checks on the result/generated inputs.
 ##' @export
-deepstate_editable_fun<-function(package_path,function_name){
-  deepstate_fun_create(package_path,function_name,sep="generation")  
-  deepstate_fun_create(package_path,function_name,sep="checks")  
-  }
+deepstate_editable_fun <- function(package_path, function_name) {
+  gen <- deepstate_fun_create(package_path, function_name, sep="generation")
+  chk <- deepstate_fun_create(package_path, function_name, sep="checks")
+  
```

Comment:
please remove empty space

Reply:
I see you removed spaces after commas... which is not what I meant, sorry for the confusion. (maybe you want to revert this commit?)
I meant please remove empty lines.
Anyway in both cases it does not really matter.
Overall the rule should be, when you commit some changes, try to respect the formatting/style nearby.

Reply:
I am sorry, I probably misunderstood. I have just reverted to the previous commit and removed the empty lines. 
Commit edf31e1